docs/user/flyout-menu.rst

```diff
@@ -8,8 +8,30 @@ you will likely notice that we embed a menu on all the documentation pages we se
 This is a way to expose the functionality of Read the Docs on the page,
 without having to have the documentation theme integrate it directly.
 
-Functionality
--------------
+There are two versions of the flyout menu:
+
+- :ref:`flyout-menu:Addons flyout menu`
+- :ref:`flyout-menu:Original flyout menu`
+
+Addons flyout menu
```

Comment:
The new flyout has some features the old one doesn't. Like the ability to sort the versions in a different way: semver, calver, custom pattern. Where those type of addon configuration should live? In this page?

Reply:
Yeah, I think we can just add a section before Customizing on line 26